tests/unit/test-date-helpers.c

```diff
@@ -32,9 +32,10 @@
  * Unit test's main function.
  */
 int
-main (int __attr_unused___ argc,
-      char __attr_unused___ **argv)
+main (int argc, char **argv)
```

Comment:
I think you can simply delete both arguments, and just use:

    main ()
    {
    }

Several similar cases below as well.